src/raft.rs

```diff
@@ -1312,6 +1314,18 @@ impl<T: Storage> Raft<T> {
                         "msg type" => ?m.get_msg_type(),
                     );
 
+                    // When judge_split_prevote, reject explicitly to make candidate exit PreCandiate early
+                    // so it will vote for other peer later.
+                    if self.judge_split_prevote
+                        && m.get_msg_type() == MessageType::MsgRequestPreVote
+                    {
```

Comment:
Seems `Follower` and `PreCandidate` are not different, both of them can (pre)vote to other peers.

Reply:
`Follower` will ignore all `PreVoteResponse`. For example, A and B split pre-votes, and C can vote for both A and B. If B is chosen, and A doesn't step down to follower, it will still start campaign when C's prevote is received.